### ISSUE TITLE
What's the status of meson port?

### ISSUE THREAD

**Issue by nacho** [centricular/fontconfig#1]
I am wondering whether to use this fork on gvsbuild. But I have a few questions though:
 - is it ready?
 - has it been tested on msvc? (if not no problem I can do it myself)
 - is there an upstream bug report about this
 - are the upstream maintainers interested on this?

@behdad any thoughts on this?

**Comment by behdad** [comment]
cc @tagoh

**Comment by tagoh** [comment]
interesting but I'm not familiar with meson at all so any patches are welcome.

**Comment by MathieuDuponchelle** [comment]
> is it ready?

I think so yes

> has it been tested on msvc? (if not no problem I can do it myself)

yes, but more testing never hurts

> is there an upstream bug report about this

Nope, there probably should be :)

> are the upstream maintainers interested on this?

hopefully so :)

**Comment by MathieuDuponchelle** [comment]
Also, we have ports of [harfbuzz](https://github.com/centricular/harfbuzz), [freetype](https://github.com/centricular/freetype2), [pixman](https://github.com/centricular/pixman) and [cairo](https://github.com/centricular/cairo) with the same status, in case that is relevant to you :)

**Comment by nacho** [comment]
@MathieuDuponchelle do you have pull requests for those projects? see that @tagoh is not opposed to this

**Comment by MathieuDuponchelle** [comment]
I need to find some time to make proposals, but that is the goal ultimately yes

**Comment by tagoh** [comment]
Yes, please open a PR at https://gitlab.freedesktop.org/fontconfig/fontconfig

**Comment by nacho** [comment]
@MathieuDuponchelle any news on this?

**Comment by MVoz** [comment]
> hmm... 
> could not repeat

**Comment by tp-m** [comment]
I'll rebase it and file a merge request at https://gitlab.freedesktop.org/fontconfig/fontconfig/

**Comment by tp-m** [comment]
Upstream fontconfig MR is here: https://gitlab.freedesktop.org/fontconfig/fontconfig/-/merge_requests/87 - just have to find someone to press the big green button :)
